spec/spec_helper.rb

```diff
@@ -44,7 +41,7 @@
 
   config.before(:suite) do
     DatabaseCleaner.strategy = :transaction
-    DatabaseCleaner.clean_with :truncation, except: %w[paises provincias subtipos tipo_productos constantes likerts pagestaticas usuarios permisos tematicas]
+    # DatabaseCleaner.clean_with :truncation, except: %w[paises provincias subtipos tipo_productos constantes likerts pagestaticas usuarios permisos tematicas]
```

Comment:
I led it there because it should work, but it doesn't. So maybe any of you know WFT could be happening..